### ISSUE TITLE
[QUESTION]  What's meaning about discounted_sum_of_advantage_scorer?

### ISSUE THREAD

**Issue by 8JasonStatham8** [takuseno/d3rlpy#275]
In the page of https://d3rlpy.readthedocs.io/en/stable/references/generated/d3rlpy.metrics.scorer.discounted_sum_of_advantage_scorer.html#
"This metrics suggests how the greedy-policy selects different actions in action-value space. If the sum of advantage is small, the policy selects actions with larger estimated action-values."
I can't understand what means of 'If the sum of advantage is small, the policy selects actions with larger estimated action-values'.
what if the sum of advantage is large, what shoud the policy do?
Looking forward your replay, thanks.

**Comment by takuseno** [comment]
By definition, if the agent thinks the agent's actions are better than actions in datasets, the sum of this advantage becomes negative.
![image](https://user-images.githubusercontent.com/5235131/232185150-47e401cf-002a-4cce-ab03-0efa9b077288.png)

This metrics indicates confidence of the agent against dataset actions.

**Comment by 8JasonStatham8** [comment]
Thanks for your reply. 
Should this score be bigger or smaller or 0？
I think if the dataset is expert behavior， the score is 0 represent the Q function is better，am I right？

**Comment by takuseno** [comment]
Yes, the smaller the score bcomes, the better the agent thinks. If it's zero, the policy actions are the exactly same as the dataset actions.
